db/schema.rb

```diff
@@ -67,6 +67,8 @@
     t.integer "role", default: 0
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.integer "uid"
+    t.string "github_token"
     t.string "gh_token"
```

Comment:
What's `uid`? Also, looks like we picked up an extra `github_token`. I would drop the first one since our app currently looks for the token in `gh_token`